sphinx/themes/basic/static/searchtools.js

```diff
@@ -163,6 +163,8 @@ const Search = {
   htmlToText: (htmlString, anchor) => {
     const htmlElement = new DOMParser().parseFromString(htmlString, 'text/html');
     htmlElement.querySelectorAll(".headerlink").forEach((el) => { el.remove() });
```

Comment:
Create a list of selectors to remove and iterate over that list instead (easier to maintain in the future)